releasenotes-builder/src/main/java/com/github/checkstyle/CliProcessor.java

```diff
@@ -183,6 +186,9 @@ private List<String> validateCli() {
         if (!cmdLine.hasOption(OPTION_RELEASE_NUMBER)) {
             result.add("Release number has not been specified!");
         }
+        if (!cmdLine.hasOption(OPTION_VALIDATE_VERSION)) {
+            result.add("version number has not been specified!");
```

Comment:
It will never be specified.
It is validation of version that is going to be released

Reply:
okay!